apps/commudle-admin/src/app/app-shared-components/hackathon-horizontal-card/hackathon-horizontal-card.component.ts

```diff
@@ -0,0 +1,55 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input, OnInit } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { NbButtonModule, NbCardModule, NbIconModule } from '@commudle/theme';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { IEvent } from 'apps/shared-models/event.model';
+import * as moment from 'moment';
+import { SharedComponentsModule } from 'apps/shared-components/shared-components.module';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { CommunitiesService } from 'apps/commudle-admin/src/app/services/communities.service';
+import { SharedDirectivesModule } from 'apps/shared-directives/shared-directives.module';
+import { faSackDollar } from '@fortawesome/free-solid-svg-icons';
+import { countries_details } from '@commudle/shared-services';
+import { EDbModels, IHackathon } from '@commudle/shared-models';
```

Comment:
we can import ICommunity and IEvent for there to, no need to import this from another path